### ISSUE TITLE
Command not found (setinregex, setinpath, setinregex, setinregex)

### ISSUE THREAD

**Issue by more-pepsi** [stefandtw/feedranger#3]
<img width="654" alt="image" src="https://user-images.githubusercontent.com/89631260/200156238-51917278-9c91-4d8e-a54d-34f83fa1135e.png">

Any idea. Ubuntu 22.04

**Comment by stefandtw** [comment]
About a year ago, ranger's `setlocal` command was replaced by `setinpath` and `setinregex`.

Your ranger version does not have this change yet and is therefore not compatible with the latest commit in feedranger.

You can use an older feedranger version.

Either checkout commit 5016ed5df70532a8d531400c9da297ee16384378
or revert commit 9cb50b9e7e6652cd2b8462947b41803b54f50cd7

**Comment by more-pepsi** [comment]
Thanks man. Pulling together that list of dependencies is hard when you first start using a tool haha. 

Are wanting pull requests? Small changes, i changed the shebang to python3 since, I don't want python2 applications calling python3, I also removed firefox. I would add any changes as options etc.

**Comment by stefandtw** [comment]
Pull requests are welcome, though one thing to keep in mind: feedranger is meant to be highly customized by the user. Changes desired by a single user should be implemented by the user for themselves first. That said, if it's something that is likely to benefit everyone, I'd be happy to accept pull requests.

Shebang is simply the same as in ranger itself. ranger and feedranger both are usable with python2 and python3. Perhaps ranger will change it at some point or distributions will drop python2.

**Comment by stefandtw** [comment]
> My ranger version is 1.9.3 thats the latest https://github.com/ranger/ranger/releases/tag/v1.9.3

True, but the release is now three years old dispite many more recent commits. Arch Linux at least has shipped packages based on commit versions. Most distributions will probably not do this. At the same time, many ranger users are also Arch users. Haven't really put any effort into making things compatible with multiple versions until now.
